### ISSUE TITLE
False positive - WITH HEADER LINE can only be used with internal table(check_syntax)

### ISSUE THREAD

**Issue by ed-holland** [abaplint/abaplint#1729]
Source:
```
TYPE-POOLS rkea1.
DATA: fieldtab TYPE rkea1_fieldtab WITH HEADER LINE.
```

Resulting in:
`zfoobar.prog.abap[6, 1]: WITH HEADER LINE can only be used with internal table(check_syntax)`

The issue seems to be that the type-pool is not created as dependency.
as such, it can not be resovled.

**Comment by ed-holland** [comment]
It might be related  to #1667 (getting old).

**Comment by larshp** [comment]
reproduced as 
![image](https://user-images.githubusercontent.com/5888506/106882052-97c56100-66de-11eb-82bb-8ea620a59e29.png)

**Comment by ed-holland** [comment]
Issue is resolved. Thanks.
